index.bs

```diff
@@ -239,7 +239,7 @@ its \[[Prototype]] an empty object, created as if by ObjectCreate(%ObjectPrototy
 
 <h4 id="clear" oldids="dom-console-clear" method for="console">clear()</h4>
 
-If possible for the environment, clear the console. Otherwise, do nothing.
+End all open <a>groups</a>, and if possible for the environment, clear the console. Otherwise, do nothing.
```

Comment:
"End all open groups" should probably be "empty the appropriate group stack" right?